packages/app/src/containers/ComponentsWindow.tsx

```diff
@@ -0,0 +1,53 @@
+import { memo } from 'react';
+import ColorSwatch from '../components/swatches/ColorSwatch';
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { GridView } from 'noya-designsystem';
+import { useApplicationState } from '../contexts/ApplicationStateContext';
+import { getSharedSwatches } from 'noya-state/src/selectors';
+import { rgbaToHex } from 'noya-colorpicker';
+
+export default memo(function ComponentsWindow() {
+  const [state, dispatch] = useApplicationState();
+
+  const items = getSharedSwatches(state);
+
+  const createSubtitle = (item: Sketch.Swatch) => {
+    if (item._class === 'swatch') {
+      const color = {
+        a: item.value.alpha,
+        r: Math.round(item.value.red * 255),
+        g: Math.round(item.value.green * 255),
+        b: Math.round(item.value.blue * 255),
+      };
+
+      return `${rgbaToHex(color)} - ${Math.round(color.a * 100)}%`;
+    }
+    return '';
+  };
+
+  return (
+    <GridView.Root onClick={() => dispatch('selectSwatch', undefined)}>
+      {items.map((item, index) => (
```

Comment:
Sketch sorts color swatches by name when presenting them in the grid - let's do the same

Reply:
I have a question about this one. I sorted them but when I add a new one should I place it at the end? and if so if I add two new ones should the previous one stay at the bottom or should it move with the name "New Color Variable" where it belongs ?

Reply:
I think new ones should be sorted too, rather than placed at the end. 

Ideally a new color would also scroll into view if needed, and the name input field would be focused automatically, but we can do that in the future

Reply:
I understand I'll leave it as it is for now. It's sorted